crates/next-core/js/src/ipc/server.ts

```diff
@@ -106,11 +106,10 @@ export function makeRequest(
       port: address.port,
       method,
       path:
-        query && Object.keys(query).length > 0
-          ? `${path}?${qs.stringify(query)}`
-          : path,
-      headers,
+        rawQuery != null && rawQuery.length > 0 ? `${path}?${rawQuery}` : path,
```

Comment:
```suggestion
        rawQuery ? `${path}?${rawQuery}` : path,
```

Reply:
My personal preference is to avoid boolean coercions in JS altogether (bringing it closer to Rust). Otherwise, it's unclear whether I meant to include the empty string in my truthy check or not.